.travis-install-tgui.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -ev
```

Comment:
I kinda wonder why this needs to be a separate file instead of part of the main install file? I guess it doesn't hurt though.

Reply:
I wanted to keep tgui separate. In install-deps there are just packages, so I thought compilation is better kept in separate one.